### DISCUSSION TITLE
Change lazer installation path

### DISCUSSION THREAD

**Discussion by xhrnca00** [ppy/osu#29785] [Ideas]
We can now change where data is stored, but I would like if you could also change the installation path of lazer itself. I try to keep my C: drive as clean as possible and try to put all applications on my F: drive instead.

Is this something that could be added to the installer? If not, what happens if I move the lazer folder manually (and re-link all shortcuts)? Will updates continue to work in the new directory, or will they put lazer back in its original place?

**Comment by peppy** [comment]
You can move it manually maybe. We will probably support portable installations in the future. But the install is tiny so you shouldn't have a huge issue even if you can't move it.
